tools/interop_matrix/client_matrix.py

```diff
@@ -211,6 +211,7 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
             ('v1.50.1', ReleaseInfo(runtimes=['go1.16'])),
             ('v1.51.0', ReleaseInfo(runtimes=['go1.16'])),
             ('v1.52.0', ReleaseInfo(runtimes=['go1.19'])),
+            ('v1.52.3', ReleaseInfo(runtimes=['go1.19'])),
```

Comment:
usually we'd only add a single patch version for a given minor release (e.g. we'd only have the latest patch from 1.52.x). I'd suggest removing 1.52.0 while adding 1.52.3.